src/index.js

```diff
@@ -374,7 +374,10 @@ class Dropzone extends React.Component {
       accept,
       disabled,
       type: 'file',
-      style: { display: 'none' },
+      style: {
+        display: 'none',
+        ...(inputProps && inputProps.style)
```

Comment:
Do you need to check for existence of `style` here? AFAIK, you can use `...` with `undefined` value?

Reply:
Yea, it works fine with undefined, I tested even in safari:

<img width="187" alt="screen shot 2018-05-25 at 15 09 47" src="https://user-images.githubusercontent.com/7217420/40545887-c55f1eb8-602d-11e8-9dec-3f34b5c021b3.png">

Reply:
Can you update the code please?

Reply:
@okonet what do you mean? Update the code how?

Reply:
Remove the `(inputProps &&` part since it's not needed. `...inputProps.style` should be enough if `inputProps` would be set to `{}` in `defaultProps`?

Reply:
oh sure thing, sorry it was not clear